src/libs/OptionsListUtils.js

```diff
@@ -159,17 +160,22 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     searchValue = '',
     showChatPreviewLine = false,
     showReportsWithNoComments = false,
+    hideReadReports = false,
+    sortByAlphaAsc = false,
 }) {
     let recentReportOptions = [];
     const pinnedReportOptions = [];
     const personalDetailsOptions = [];
 
     const reportMapForLogins = {};
-    const orderedReports = lodashOrderBy(reports, [
+    let orderedReports = lodashOrderBy(reports, [
```

Comment:
It seems kind of wasteful to reassign this and have to filter reports twice (they can be some huge number)

Should be possible to do something like this and avoid the double sort?

```js
const sortProperty = [sortByAlphaAsc ? 'reportName' : blah blah];
const sortDirection = [sortByAlphaAsc ? 'asc' : 'desc'];
const orderedReports = lodashOrderBy(reports, sortProperty, sortDirection)
```

Reply:
👍  I like this idea.